indexer/categories_holder.hpp

```diff
@@ -65,6 +65,7 @@ class CategoriesHolder
 public:
   static int8_t const kEnglishCode;
   static int8_t const kUnsupportedLocaleCode;
+  static uint8_t constexpr kMaxSupportedLocaleIndex = 30;
```

Comment:
You can initialize in header other constants, right?

Reply:
done